README.md

```diff
@@ -35,6 +35,17 @@ python edit_app.py
 ![Edit app](https://github.com/timothybrooks/instruct-pix2pix/blob/main/imgs/edit_app.jpg?raw=true)
 
 _(For advice on how to get the best results by tuning parameters, see the [Tips](https://github.com/timothybrooks/instruct-pix2pix#tips) section)._
+### Quickstart via [Imaginairy](https://github.com/brycedrennan/imaginAIry#-edit-images-with-instructions-alone-by-instructpix2pix)
+**(alternative installation method)**
+```bash
+pip install imaginairy --upgrade
+# edit a single image and make a gif showing the difference
+aimg edit any-image.jpg --gif --edit "turn him into a cyborg" 
```

Comment:
Executing this will trigger an error:

```text
Usage: aimg edit [OPTIONS] PATH|URL [PROMPT_TEXTS]...
Try 'aimg edit --help' for help.

Error: No such option: --edit (Possible options: --height, --seed, --width)
```

How about this?

```suggestion
aimg edit any-image.jpg --gif "turn him into a cyborg"
```

Reply:
Whoops! thank you - fixed now